scripts/linters/general_purpose_linter.py

```diff
@@ -216,7 +216,10 @@
             'core/templates/filters/translate.pipe.spec.ts',
             'core/templates/components/ck-editor-helpers/' +
             'ck-editor-copy-content-service.spec.ts',
-            'core/templates/tests/unit-test-utils.ts'),
+            'core/templates/tests/unit-test-utils.ts',
+            'core/templates/directives/mathjax.directive.ts',
+            'extensions/objects/templates/' +
+            'math-expression-content-editor.component.ts'),
```

Comment:
Why are we adding more things here? Also, why do we have a lint check for this if we can use it in multiple places? (Just wanted to check what and why we are blocking here!)

Reply:
The old code used jquery to do the innerHTML manipulation. In jquery, we just do `.html()`. For this reason, it was not caught by the lint checks. This innerHTML is need for MathJaz to be able to create math expressions in CKeditor. Does this answer your question?

Reply:
The old code used jquery to do the innerHTML manipulation. In jquery, we just do `.html()`. For this reason, it was not caught by the lint checks. This innerHTML is need for MathJaz to be able to create math expressions in CKeditor. Does this answer your question?

Reply:
Overall this check is made for avoiding jquery calls?

Reply:
Technically this check is supposed to stop us from using innerHTML in HTML files. But this check only looks for innerHTML in ts files. 😂